lib/activitylog/hooks/logger.js

```diff
@@ -5,7 +5,7 @@ module.exports = db => {
   return model => {
     return ActivityLog.query(db).insert({
       caseId: model.id,
-      changedBy: get(model, 'meta.user.id'),
+      changedBy: get(model, 'meta.user.profile.id'),
```

Comment:
How feasible is it to add a middleware in workflow that does `req.user = req.user.profile`?

I'm wary about leaking implementation detail (user object structure) into the framework, but also if it's in any way a pain then I'll live with this.

Reply:
It's a pain. I'll live with it.